### PR TITLE
fixed according to PEP8

### PR DESCRIPTION
**Pull request by VladyslavHnatchenko**
Fixed according to PEP8.
Please check.

Thank you.

### CODE REVIEW

**Comment by hamishwillee** [comment]
Thanks @VladyslavHnatchenko 
What tool did you use for PEP8 compliance, or was this manual? Did you test updated code - if so, how?

File: .gitignore [hunk 1]
```diff
@@ -94,3 +94,5 @@ ENV/
 
 # Rope project settings
 .ropeproject
+
+.idea
```
Comment (hamishwillee) [inline .gitignore]: What is rope project? Why is .idea needed?

Comment (VladyslavHnatchenko) [reply .gitignore]: Hi @hamishwillee!

This is manual fixing according to PEP8(https://www.python.org/dev/peps/pep-0008/#blank-lines).
Yes, I was tested and all working is fine with PyCharm IDE(https://www.jetbrains.com/pycharm/).
.idea not needed, this my local directory. When I create some project on my local machine with PyCharm, this directory .idea created.

Thank you!

Comment (VladyslavHnatchenko) [reply .gitignore]: Yes, it's done.
Thank you!

Comment (hamishwillee) [reply .gitignore]: Thank you. I'll do quick test and merge shortly (probably tomorrow).

**Comment by hamishwillee** [comment]
Tested and merged. Thanks very much @VladyslavHnatchenko !

**Comment by VladyslavHnatchenko** [comment]
Thank you very much @hamishwillee!
Have a good day!

**Pull request merged**